### PR TITLE
Swallow "AlreadyMember" error from zerokit

### PR DESCRIPTION
**Pull request by sarahESL**
This error happens when grantPermission with a grantee happens for the second time.
This is error is received from zerokit_sdk. Current decision is to treat this
error as if the grantPermission has succeeded.

### CODE REVIEW

File: test/services/zerokitAdapterTest.js [hunk 1]
```diff
@@ -371,6 +371,9 @@ describe('zerokitAdapter', () => {
 				expect(userRouteVerifyShareAndGrantPermissionStub).to.be.calledOnce;
 				UserService.user = undefined;
 				done();
+			})
+			.catch((err) => {
```
Comment (Glumli) [inline test/services/zerokitAdapterTest.js]: This should not be here ;)
But would the idea to change the test file is good. Please write a test for this case :D

Comment (sarahESL) [reply test/services/zerokitAdapterTest.js]: where should it be then? because if I remove this, I mean the catch part, the test would not pass!

Comment (Glumli) [reply test/services/zerokitAdapterTest.js]: Then there is something wrong in the test ;)
I would add a test in which, you stub the zkit method to return the error and then check wether it still passes

File: src/services/ZeroKitAdapter.js [hunk 2]
```diff
@@ -199,7 +199,11 @@ class ZeroKitAdapter {
 				return userRoutes.verifyShareAndGrantPermission(UserService.getUserId(),
 					grantee.id, operationId);
 			})
-			.then(() => operationId);
+			.then(() => operationId)
```
Comment (Glumli) [inline src/services/ZeroKitAdapter.js]: Sarah didn't want to return the operationId anymore. The rest looks good.

File: src/services/ZeroKitAdapter.js [hunk 3]
```diff
@@ -196,10 +196,9 @@ class ZeroKitAdapter {
 			.then(zeroKit => zeroKit.shareTresor(tresorId, grantee.zeroKitId))
 			.then((response) => {
```
Comment (Glumli) [inline src/services/ZeroKitAdapter.js]: The operationId variable is not necessary anymore.
.then(operationId => userRoutes
	.verifyShareAndGrantPermission(UserService.getCurrentUser().id, grantee.id, operationId))

**Pull request merged**